Source/Upstream: Yes, in v238-stable branch

From 64eafeaf75c2953e1ed136e4c69e14cb392d2f05 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Wed, 14 Mar 2018 14:06:50 +0900
Subject: [PATCH 39/51] man: mention that 'status' is the default command for
 `hostnamectl`

(cherry picked from commit a456324fefec12ac929fd844bcec243b657d7b1f)
---
 man/hostnamectl.xml | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/man/hostnamectl.xml b/man/hostnamectl.xml
index fa70d5cc6..9bfead6d2 100644
--- a/man/hostnamectl.xml
+++ b/man/hostnamectl.xml
@@ -121,6 +121,10 @@
       <xi:include href="standard-options.xml" xpointer="help" />
       <xi:include href="standard-options.xml" xpointer="version" />
     </variablelist>
+  </refsect1>
+
+  <refsect1>
+    <title>Commands</title>
 
     <para>The following commands are understood:</para>
 
@@ -128,9 +132,8 @@
       <varlistentry>
         <term><command>status</command></term>
 
-        <listitem><para>Show current system
-        hostname and related
-        information.</para></listitem>
+        <listitem><para>Show current system hostname and related information. If no command is specified,
+        this is the implied default.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-- 
2.16.3

